### ISSUE TITLE
Click and scroll action over widgets

### ISSUE THREAD

**Issue by UtkarshVerma** [arcan1s/awesome-widgets#140]
Is it possible to get clicking and scrolling action over widget as it was there in i3blocks? For example, I have a script which displays the currently playing song.
I want to skip to the next song by right-clicking on it etc. In i3blocks, I used to bind a command to the click.
Is it possible to get the same behaviour here?

**Comment by arcan1s** [comment]
not really. The main issue to implement this is that widget is a single text label, not different blocks, which makes hard to handle clicking on different text elements

**Comment by arcan1s** [comment]
for now i'm going to put this task to backlog, sorry. I will try to figure out the way how I can handle it

**Comment by UtkarshVerma** [comment]
Okay, no problem for me in waiting.
